scripts/setup/create_multinode.sh

```diff
@@ -58,8 +58,13 @@ server_exec() {
 
 common_init() {
     internal_init() {
-        server_exec $1 "git clone --branch=$VHIVE_BRANCH https://github.com/ease-lab/vhive"
-        server_exec $1 "cd; ./vhive/scripts/cloudlab/setup_node.sh $OPERATION_MODE"
+        server_exec $1 "git clone --branch=$VHIVE_BRANCH https://github.com/wanghanchengchn/vHive.git vhive"
+
+        server_exec $1 'sudo wget -q https://github.com/mikefarah/yq/releases/download/v4.27.3/yq_linux_amd64 -O /usr/bin/yq && sudo chmod +x /usr/bin/yq'
```

Comment:
Do we need to install on every node? I think that yamls that we apply need to be used only on master node.